File: DisplaFrontend/src/app/model/comprobanteIVA.ts
import { ComprobanteProveedor } from "./comprobanteProveedor";

export interface ComprobanteIVA {
    Id: number,
    IdComprobanteProveedor: number,
    Alicuota: number,
    MontoIva: number,
    Neto: number,

    IdComprobanteProveedorNavigation: ComprobanteProveedor,

}

File: DisplaBackend/DisplaBackend/Models/ComprobanteIva.cs
using System;
using System.Collections.Generic;

namespace DisplaBackend.Models
{
    public partial class ComprobanteIva
    {
        public int Id { get; set; }
        public int IdComprobanteProveedor { get; set; }
        public decimal Alicuota { get; set; }
        public decimal MontoIva { get; set; }
        public decimal Neto { get; set; }

        public virtual ComprobanteProveedor IdComprobanteProveedorNavigation { get; set; }
    }
}
